src/games/strategy/triplea/delegate/AirBattle.java

```diff
@@ -761,14 +762,8 @@ private void notifyCasualties(final GUID battleID, final IDelegateBridge bridge,
       public void run() {
         try {
           getRemote(firingPlayer, bridge).confirmEnemyCasualties(battleID, "Press space to continue", hitPlayer);
-        } catch (final ConnectionLostException cle) {
-          // somone else will deal with this
-          // System.out.println(cle.getMessage());
-          // cle.printStackTrace(System.out);
-        } catch (final GameOverException e) {
-          // ignore
         } catch (final Exception e) {
-          // ignore
+          ClientLogger.logQuietly(e);
```

Comment:
The others at least printed stack traces. This one is just bad and should not stand. I don't think it's worth removing work so we can have consistency. Even if I did make the updates to fix all the ex.printStacktraces, that would be consistent only for the happens that I happened to have touched. We have a lot more examples of that problem, so even fixing all the ones called out here would not be consistent.